### PR TITLE
remove_ExplorerBrowseQueriesPane

### PR DESCRIPTION
**Pull request by hardiknai-techm**
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/889?feature.someFeatureFlagYouMightNeed=true)

### CODE REVIEW

File: src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx [hunk 1]
```diff
@@ -529,7 +549,17 @@ function createManageGitHubAccountButton(container: Explorer): CommandButtonComp
...
-    onCommandClick: () => container.openGitHubReposPanel(label),
+    onCommandClick: () =>
+      useSidePanel
+        .getState()
+        .openSidePanel(
+          label,
+          <GitHubReposPanel
+            explorer={container}
+            gitHubClientProp={container.notebookManager.gitHubClient}
+            junoClientProp={undefined}
```
Comment (southpolesteve) [inline src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx:560]: Is this correct? Passing `undefined`? If so we should change the type of junoClientProp to be optional

File: src/Explorer/Notebook/NotebookManager.tsx [hunk 2]
```diff
@@ -170,7 +180,17 @@ export default class NotebookManager {
...
-        () => this.params.container.openGitHubReposPanel("Connect to GitHub"),
+        () =>
+          useSidePanel
+            .getState()
+            .openSidePanel(
+              "Connect to GitHub",
+              <GitHubReposPanel
+                explorer={this.params.container}
+                gitHubClientProp={this.params.container.notebookManager.gitHubClient}
+                junoClientProp={undefined}
```
Comment (southpolesteve) [inline src/Explorer/Notebook/NotebookManager.tsx:191]: Same comment as above. Unlcear why we are passing undefined here

File: src/Explorer/Tree/ResourceTreeAdapter.tsx [hunk 3]
```diff
@@ -611,7 +613,17 @@ export class ResourceTreeAdapter implements ReactAdapter {
...
-        onClick: () => this.container.openGitHubReposPanel("Manage GitHub settings"),
+        onClick: () =>
+          useSidePanel
+            .getState()
+            .openSidePanel(
+              "Manage GitHub settings",
+              <GitHubReposPanel
+                explorer={this.container}
+                gitHubClientProp={this.container.notebookManager.gitHubClient}
+                junoClientProp={undefined}
```
Comment (southpolesteve) [inline src/Explorer/Tree/ResourceTreeAdapter.tsx:624]: Same comment as above

File: src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx (hunk 1 above)
Comment (hardiknai-techm) [reply src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx:560]: done

File: src/Explorer/Tree/ResourceTreeAdapter.tsx (hunk 3 above)
Comment (hardiknai-techm) [reply src/Explorer/Tree/ResourceTreeAdapter.tsx:604]: done

File: src/Explorer/Notebook/NotebookManager.tsx (hunk 2 above)
Comment (hardiknai-techm) [reply src/Explorer/Notebook/NotebookManager.tsx:191]: done

**Comment by southpolesteve** [comment]
@hardiknai-techm can you fix merge conflicts with master?

**Comment by hardiknai-techm** [comment]
> 
> 
> @hardiknai-techm can you fix merge conflicts with master?

resolved

**Review by victor-meng — APPROVED** [review]

File: src/Explorer/Notebook/NotebookManager.tsx [hunk 4]
```diff
@@ -88,7 +89,16 @@ export default class NotebookManager {
       this.gitHubClient.setToken(token?.access_token);
       if (this?.gitHubOAuthService.isLoggedIn()) {
         useSidePanel.getState().closeSidePanel();
-        this.params.container.openGitHubReposPanel("Manager GitHub settings", this.junoClient);
+        useSidePanel
+          .getState()
+          .openSidePanel(
+            "Manager GitHub settings",
```
Comment (southpolesteve) [inline src/Explorer/Notebook/NotebookManager.tsx:95]:
```suggestion
            "Manage GitHub settings",
```

**Pull request merged**